.github/scripts/mega-app-build.sh

```diff
@@ -46,7 +46,7 @@ while [[ $# -gt 0 ]]; do
         PKG_MANAGER=$2
         shift
         ;;
-    -@ | --platform)
+    -A | --platform)
```

Comment:
I don't see `-p` being used, can we use that?

Reply:
Though `-p` is not used in mega-app-build.sh, it's used in mega-app-install.sh. I'd be better to make them consistent since their the same serial of commands.

Reply:
That makes sense. I'm good with `-A`, not too concerned with it!

Reply:
Yea, true. macOS has that, but linux doesn't